src/py21cmfast/src/JSC.h

```diff
@@ -0,0 +1,571 @@
+// Things needed for Radio/PBH projects
+// Everything is in SI unit unless specified otherwise
+
+#include "Tables.h"
+#include "Hawking_Radiation.h"
+
+// ---- Knobs and Switches ----
+// nu0 is degenerate with fR so no reason to leave this as a param
+#define astro_nu0 0.15 // in GHz
+
+// Debuging options, print out gas temp and SFRD box, I am gonna keep these
+#define print_SFRD_box 1
+#define Reset_Radio_Temp_HMG 0
+#define debug_mode 1
```

Comment:
Might not need this since we have LOG_DEBUG etc.

Reply:
It might actually be more convenient to keep the print_SFRD_box feature, this is used to print out huge arrays to a file. Using other original debug features (e.g LOG_DEBUG) will also turn on a bunch of other debug infos that are irrelevant for the new features